OpenRA.Game/Map/Map.cs

```diff
@@ -1407,5 +1408,10 @@ public string Translate(string key, IDictionary<string, object> args = null)
 
 			return modData.Translation.GetString(key, args);
 		}
+
+		public void Dispose()
+		{
+			Sequences.Dispose();
+		}
```

Comment:
This feels like it shouldn't belong in the `Map.cs` class given that it is purely related to sequences (understand there is scope to add more to the Dispose function, but that isn't the case currently). It will need to be moved out of `Map.cs` anyway once we start using [specialised Map formats](https://github.com/OpenRA/OpenRA/pull/20582).

Reply:
`Map` owns `Sequences`, so it is its job to dispose it!